app/controllers/v0/virtual_agent/virtual_agent_appeal_controller.rb

```diff
@@ -7,7 +7,9 @@ module VirtualAgent
     class VirtualAgentAppealController < AppealsBaseController
       def index
         # @appeal_status_description
-
+        user_info = { first_name: current_user.first_name, last_name: current_user.last_name, ssn: current_user.ssn,
+                      icn: current_user.icn }
+        VirtualAgentStoreUserInfoJob.perform_async(user_info, 'appeals', {})
```

Comment:
I'm wondering if you will want to encrypt the user_info arguments here. In its current state, the (PII.. mainly concerned about SSN) args will be stored in redis as unencrypted. There's several examples in the codebase where encrypted args are used with Sidekiq jobs via [KMS + Lockbox](https://github.com/department-of-veterans-affairs/vets-api/blob/master/modules/covid_research/app/services/covid_research/volunteer/form_crypto_service.rb#L9)

Reply:
So long as we can decrypt the SSN upon reading from the table, we can do encryption

Reply:
Done!

Reply:
@LindseySaari any chance you could approve this?